src/datumaro/components/annotation.py

```diff
@@ -83,6 +84,11 @@ class Annotation:
     # single object. The value of 0 means there is no group.
     group: int = field(default=NO_GROUP, validator=default_if_none(int))
 
+    # obeject identifier over the multiple items
+    # e.g.) in a video, person 'A' could be annotated on the multiple frame images
+    #   the user could assign >=0 value as id of person 'A'.
+    object_id: int = field(default=NO_OBJECT_ID, validator=default_if_none(int))
```

Comment:
This can be a stupid question, but can we use 'id' but change its default value to -1, instead of using object_id?

Reply:
I recognized the purpose of two variables are different. I'd like to withdraw my previous comment.

Reply:
This is a good point. Wonju and I discuss what if we could use `id` to distinguish the objects over the frames.
But we conclude that `id` better be remains as it is since `id` has its own features.